app/controllers/tas_controller.rb

```diff
@@ -95,4 +95,17 @@ def upload_ta_list
   def user_params
     params.require(:user).permit(:user_name, :last_name, :first_name)
   end
+
+  before_action :set_assignment_ta, only: [:refresh_graph]
```

Comment:
Don't make a separate method; put the two lines inside the `refresh_graph` method.